### ISSUE TITLE
nightscout@ranneft: Not updating local installation

### ISSUE THREAD

**Issue by inventor96** [linuxmint/cinnamon-spices-applets#3406]
```
 * Applet version/Build date: 0.0.1
 * Cinnamon version: 4.6.7
 * Distribution: Mint 20
 * Graphics hardware *and* driver used:
    * Intel Corporation 4th Gen Core Processor Integrated Graphics Controller (not sure on the driver)
    * NVIDIA Corporation GK106GLM [Quadro K2100M] (nvidia-driver-390)
 * Snippet or attachment of ~/.cinnamon/glass.log
    * Doesn't exist
 * Snippet or attachment of ~/.xsession-errors
    $ grep  nightscout .xsession-errors
    Cjs-Message: 16:15:47.837: JS LOG: [LookingGlass/info] Installing settings for nightscout@ranneft
    Cjs-Message: 16:15:47.837: JS LOG: [LookingGlass/info] Settings successfully installed for nightscout@ranneft
    Cjs-Message: 16:15:47.845: JS LOG: [LookingGlass/info] Loaded applet nightscout@ranneft in 52 ms
    Cjs-Message: 16:16:38.143: JS LOG: [LookingGlass/info] Reloading applet: nightscout@ranneft/31

 ```

**Notify author of applet**
@ImmRanneft 

**Issue**
Even after updating the cache from the Applets window, and uninstalling + reinstalling the Nightscout plugin, I'm still getting old code. After #3400 and #3401 were merged, I figured I'd revert to the distributed version instead of using my own code, but I'm still getting version 0.0.1 of the nightscout applet.

**Steps to reproduce**
1. Refresh the cache in the Applets window
2. Install the Nightscout plugin
3. Enable it
4. Right click on the plugin on the panel, and select "About..."
5. Note the version number

**Expected behaviour**
I would expect the version number to be 0.1.0.

**Other information**
The actual behavior from the steps above is that I get version number 0.0.1. To further confirm that I'm not getting the updated code, I'm encountering the bugs that #3400 and #3401 are supposed to fix.

**Comment by NikoKrause** [comment]
This is a bug in Cinnamon itself. It has been fixed in Cinnamon 4.8. It's planned to backport this fix to older Cinnamon versions.

**Comment by inventor96** [comment]
I just updated to Mint 20.1, which includes Cinnamon 4.8.5, and I can confirm that I was able to update the plugin. It didn't flag it as having an update available, so I uninstalled it, and then reinstalled it. I looked at the local copy of the code, and it had the latest commit.
